examples/with-jest/index.test.js

```diff
@@ -0,0 +1,19 @@
+import {matcher, serializer} from 'jest-glamor-react'
+import React from 'react'
+import {shallow, render, mount} from 'enzyme'
+import {Wrapper, Title} from './index'
+
+expect.addSnapshotSerializer(serializer)
```

Comment:
Let's not forget the comment here. Or actually, I think it would be better to do the setup file because that's what people should do in real applications.